ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/QBFTGenesisFileModule.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Hyperledger Besu Contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.evmtool;
+
+import org.hyperledger.besu.config.GenesisConfigOptions;
+import org.hyperledger.besu.consensus.common.bft.BftBlockHeaderFunctions;
+import org.hyperledger.besu.consensus.qbft.QbftExtraDataCodec;
+import org.hyperledger.besu.consensus.qbft.QbftProtocolSchedule;
+import org.hyperledger.besu.ethereum.core.BlockHeaderFunctions;
+import org.hyperledger.besu.ethereum.core.PrivacyParameters;
+import org.hyperledger.besu.ethereum.mainnet.ProtocolSchedule;
+import org.hyperledger.besu.evm.internal.EvmConfiguration;
+
+import javax.inject.Named;
+
+class QBFTGenesisFileModule extends GenesisFileModule {
+  final QbftExtraDataCodec bftExtraDataEncoder = new QbftExtraDataCodec();
+
+  QBFTGenesisFileModule(final String genesisConfig) {
+    super(genesisConfig);
+  }
+
+  @Override
+  ProtocolSchedule provideProtocolSchedule(
+      final GenesisConfigOptions configOptions,
+      @Named("RevertReasonEnabled") final boolean revertReasonEnabled) {
+    return QbftProtocolSchedule.create(
```

Comment:
Should be able to use the simpler create constructor here, i.e. QbftProtocolSchedule.create(configOptions, bftExtraDataEncoder)